app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+import 'components/Watchlist/TitleCard/cardStyle.scss';
+import TitleDialog from './TitleDialog';
+
+export default function TitleCard({ titles }) {
+  const [open, setOpen] = useState(false);
+
+  const handleOpenDialog = () => {
+    setOpen(true);
+  };
+
+  const handleCloseDialog = () => {
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <Card>
+        <CardActionArea onClick={handleOpenDialog}>
+          <CardMedia
+            className='media'
+            style={{ paddingTop: '60%' }}
```

Comment:
This seems to work with `padding` specified in `.scss`. Is there any reason to inline it?

Reply:
man neveikia, tada paveiksliuku nerodo visiskai. Rasiau vakar del to i github. Jei uzdedu ant cardmedia className="media" ir parasau scss faile 
     .media{
    padding-top: '60%'
}
tai nerodo, o inline stilium- jau rodo foto.